apps/src/netsim/NetSimShardSelectionPanel.html.ejs

```diff
@@ -0,0 +1,37 @@
+<%
+  var i18n = require('../../locale/current/netsim');
+%>
+<div class="content-wrap">
+  <div class="field-box display-name-control">
+    <label for="netsim_lobby_name"><%= i18n.myName() %></label>
+    <input id="netsim_lobby_name" type="text" value="<%= displayName %>" <% if (displayName.length > 0) { %>disabled<% } %> />
```

Comment:
Looks like in our doc we recommend https://github.com/thoughtbot/guides/tree/master/style/sass which suggests not using IDs for styling at least (no mention of JS selection, though).

In any case, I'm pretty sure across most of CSS-land dashes are preferred. The [Google style guide](http://google-styleguide.googlecode.com/svn/trunk/htmlcssguide.xml) suggests dashes for IDs